spec/unit/software_spec.rb

```diff
@@ -505,7 +505,7 @@ module Omnibus
       let(:manifest_entry) {Omnibus::ManifestEntry.new("software", {locked_version: "1.2.8", locked_source: a_source})}
```

Comment:
Would adding a test that utilizes a `ManifestEntry` that includes a license be of any value? I'm guessing not...